### PR TITLE
versionning with macro

### PR DESCRIPTION
**Pull request by jsaintvanne**
Add a macro for tool version in macro.xml and change names of tools like "version+metaMSVersion+galaxyVersion"

### CODE REVIEW

**Comment by jsaintvanne** [comment]
I don't know why tests are not good... I just changed the versions...

File: tools/metaMS_plot/metaMS_plot.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
- <tool id="metams_plot" name="metaMS.plot" version="1.0.0">
+ <tool id="metams_plot" name="metaMS.plot" version="1.0.0+metaMS@TOOL_VERSION+galaxy0">
```
Comment (lecorguille) [inline tools/metaMS_plot/metaMS_plot.xml]: `version="@TOOL_VERSION@+galaxy0"`
(il manque un `@` aussi ici)
Autant en profiter que cet outil n'est pas trop haut :)

**Review by lecorguille — CHANGES_REQUESTED** [review]

File: tools/macros/macros.xml [hunk 2]
```diff
@@ -10,7 +12,7 @@
 
     <xml name="requirements">
         <requirements>
-            <requirement type="package" version="1.22.0">bioconductor-metams</requirement>
+            <requirement type="package" version="@TOOL_VERSION">bioconductor-metams</requirement>
```
Comment (lecorguille) [inline tools/macros/macros.xml]: Il manque un `@`

**Review by lecorguille — CHANGES_REQUESTED** [review]

File: tools/metaMS_plot/metaMS_plot.xml (hunk 1 above)
Comment (jsaintvanne) [reply tools/metaMS_plot/metaMS_plot.xml]: ha donc les outils wrapper maintenant s'appellleront "version du package + galaxy version du wrapper" si je comprends bien ?

File: tools/macros/macros.xml (hunk 2 above)
Comment (jsaintvanne) [reply tools/macros/macros.xml]: boulette sorry !

**Comment by lecorguille** [comment]
@jsaintvanne, @yguitton , there are difference in the outputs:
https://travis-ci.org/github/workflow4metabolomics/metaMS/jobs/680987115#L1620-L1632

It seems that double spaces have been replaced by simple space !?

**Comment by lecorguille** [comment]
Can you push good part of your help and the citations in the macro ?
https://github.com/workflow4metabolomics/metaMS/pull/45/files#diff-5351b461c118f9203cae1d0fb2aa9455R128-R142

File: tools/macros/macros.xml [hunk 3]
```diff
@@ -77,4 +77,34 @@
         #end if
 	</token>
 
+    <token name="@HELP_AUTHORS@">
```
Comment (lecorguille) [inline tools/macros/macros.xml:80]: You can also add the `<![CDATA[ ... ]]>` things here

Comment (lecorguille) [reply tools/macros/macros.xml:80]: And around your `@COMMAND` token too

**Review by lecorguille — CHANGES_REQUESTED** [review]

File: tools/macros/macros.xml (hunk 3 above)
Comment (jsaintvanne) [reply tools/macros/macros.xml:80]: Ho ok  it is not done in your macro of xcms and all so I did it like this :) 
So I can add `<![CDATA[ my_things ]]>` in my macro directly ? Even if I put it in my wrapper also ?

**Review by lecorguille — APPROVED** [review]

**Pull request merged**